### PR TITLE
Fixed missing arrow for neural_net.png

### PR DESCRIPTION
**Pull request by xuansontrinh**
Fixed missing arrow for `neural_net.png` used in file `slides-mc-softmax-regression.tex`

### CODE REVIEW

**Comment by xuansontrinh** [comment]
[Fix missing arrow in neural net image](https://trello.com/c/4AQI6D6n/88-fix-missing-arrow-in-neural-net-image)

**Review by ludwig-max — APPROVED** [review]
Great, thanks @xuansontrinh !

**Review by ludwig-max — APPROVED** [review]
